functorch/test/test_ops.py

```diff
@@ -1516,6 +1516,7 @@ def fn(input, weight, bias):
         skip('ldexp', dtypes=(torch.float32,), device_type='cpu'),  # fails on all but mac
         skip('__rmatmul__'),  # flaky needs investigation
         skip('matmul'),  # flaky needs investigation
+        skip('nn.functional.conv3d', dtypes=(torch.float32,)),
```

Comment:
Can you please add a comment regarding the skip.

Reply:
Hi, @kshitij12345! Sure, Added the comments for all cases. Please take another look. Thanks!